onnxscript/version_converter/_adapter_lib.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Stores all operator schema related changes between opsets"""
+
+from onnxscript.version_converter.version_converter import (
+    adapter_compatible,
+    adapter_axis_attr_to_input,
+    adapter_gridsample_19_20,
+    adapter_groupnormalization_20_21,
+)
+
+
+_ADAPTERS_18_19 = {
+    "Equal": adapter_compatible,
```

Comment:
Minor nit: I think it would be better to not have to register an adapter for compatible extensions, since it is basically an identity operation.

I can see a value in explicitly documenting that a particular opset-update is backwards-compatible (to catch the case where we forget to register an adapter). We could do that using a separate set of all compatible ops.